src/xunit.analyzers/AssertEqualShouldNotBeUsedForCollectionSizeCheck.cs

```diff
@@ -32,8 +32,8 @@ public class AssertEqualShouldNotBeUsedForCollectionSizeCheck : AssertUsageAnaly
         protected override void Analyze(SyntaxNodeAnalysisContext context, InvocationExpressionSyntax invocation, IMethodSymbol method)
         {
             if (method.Parameters.Length != 2 ||
-                !method.Parameters[0].Type.SpecialType.Equals(SpecialType.System_Int32) ||
-                !method.Parameters[1].Type.SpecialType.Equals(SpecialType.System_Int32))
+                method.Parameters[0].Type.SpecialType != SpecialType.System_Int32 ||
```

Comment:
Is this change necessary? I'd prefer to review as small a changeset as possible.

Reply:
Not strictly necessary, so I'll revert it.